apps/src/music/soundsPanel.module.scss

```diff
@@ -0,0 +1,93 @@
+@import "color.scss";
+@import "./theme/tokens-dark";
+
+.soundsPanel {
+  user-select: none;
+}
+
+%folder {
+  margin-top: 10px;
+  background-color: $gray-900;
+  overflow: hidden;
+  border-radius: 4px;
+  padding: 6px;
+}
+
+.folderFirst {
+  @extend %folder;
+  margin-top: 0;
+}
+
+.folder {
+  @extend %folder;
+  margin-top: 6px;
+}
+
+.folderName {
+  font-size: 18px;
+  padding: 6px 0;
+  margin-bottom: 6px;
+  border-bottom: solid 1px $gray-800;
+  margin-right: 12px;
+}
+
+%sound-row {
+  overflow: hidden;
+  padding: 4px 0;
+  &:hover {
+    background-color: $gray-800;
+  }
+}
+
+.soundRow {
+  @extend %sound-row;
+}
+
+.soundRowSelected {
+  @extend %sound-row;
+  background-color: $gray-700;
+}
```

Comment:
Not sure if it would make a huge difference, but to avoid the placeholder class, you could do something like
```
.soundRow {
   overflow: hidden;
   padding: 4px 0;
   &:hover {
      background-color: $gray-900
   }

   &Selected {
      background-color: $gray-700
   }
}
```
here, which also would let you use `classNames` in the component, like `className={classNames(styles.soundRow, isSelected && styles.soundRowSelected)}`

Reply:
Yes, nice!